<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabGeneral">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabProductos">
     <attribute name="title">
      <string>Productos</string>
     </attribute>
     <widget class="QLineEdit" name="tboxSearch">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>661</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Buscar:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNew">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tboxCode">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>631</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tboxName">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>631</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblProd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Codigo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Precio</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Unidades</string>
       </property>
      </column>
     </widget>
     <widget class="QPlainTextEdit" name="tboxDescription">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>380</y>
        <width>631</width>
        <height>81</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>95</width>
        <height>108</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Codigo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Precio:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Nombre:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Descripcion:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>320</y>
        <width>631</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="dspinPrice"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Unidades:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinUnits"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabClientes">
     <attribute name="title">
      <string>Clientes</string>
     </attribute>
     <widget class="QPushButton" name="btnDeleteClient">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblClients">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>461</width>
        <height>351</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Facturas</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="tboxSearchClient">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>391</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNewClient">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveClient">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Buscar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tboxCode_2">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>599</y>
        <width>631</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>461</width>
        <height>58</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Nit:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tboxClientNit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Nombre:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="tboxClientName"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Facturas Asociadas:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNuevaFactura">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>400</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Nueva Factura</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEliminarFactura">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>460</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Factura</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listboxFacturas">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>261</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFacturas">
     <attribute name="title">
      <string>Facturas</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Reportes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCarga_Masiva">
    <property name="title">
     <string>Json</string>
    </property>
    <addaction name="actionProductos_2"/>
    <addaction name="actionClientes_2"/>
   </widget>
   <widget class="QMenu" name="menuGraficas">
    <property name="title">
     <string>Graficas</string>
    </property>
    <addaction name="actionProductos_3"/>
    <addaction name="actionClientes_3"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuCarga_Masiva"/>
   <addaction name="menuGraficas"/>
  </widget>
  <action name="actionProductos">
   <property name="text">
    <string>Productos</string>
   </property>
  </action>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionProductos_2">
   <property name="text">
    <string>Cargar Productos</string>
   </property>
  </action>
  <action name="actionClientes_2">
   <property name="text">
    <string>Cargar Clientes</string>
   </property>
  </action>
  <action name="actionProductos_3">
   <property name="text">
    <string>Ilustrar Productos</string>
   </property>
  </action>
  <action name="actionClientes_3">
   <property name="text">
    <string>Ilustrar Clientes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tboxSearch</tabstop>
  <tabstop>tblProd</tabstop>
  <tabstop>tboxCode</tabstop>
  <tabstop>dspinPrice</tabstop>
  <tabstop>spinUnits</tabstop>
  <tabstop>tboxName</tabstop>
  <tabstop>tboxDescription</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>tboxSearchClient</tabstop>
  <tabstop>tblClients</tabstop>
  <tabstop>listboxFacturas</tabstop>
  <tabstop>tboxClientNit</tabstop>
  <tabstop>tboxClientName</tabstop>
  <tabstop>btnNewClient</tabstop>
  <tabstop>btnSaveClient</tabstop>
  <tabstop>btnDeleteClient</tabstop>
  <tabstop>btnNuevaFactura</tabstop>
  <tabstop>btnEliminarFactura</tabstop>
  <tabstop>tabGeneral</tabstop>
  <tabstop>tboxCode_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
